README.md

```diff
@@ -1 +1,66 @@
-# React-tutorial
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# To-Do-List with React
+
+In this project I have built a to-do-list where you can add tasks and remove tasks from.
+
+## Built With
+
+- HTML
+- CSS
+- Javascript
+- React
+
+
+## Getting Started
+
+To successfully utilize this project first of all you need to clone or download the repository. Next, open the index.html file to check the website’s main page. If you want to add or edit the files you can do that by simply opening the file folder in VS code.
```

Comment:
Kindly update the technologies list and your instructions considering you're working with React.